src/main/java/org/mskcc/cbio/oncokb/web/rest/AccountResource.java

```diff
@@ -105,6 +106,7 @@ public AccountResource(UserRepository userRepository, UserService userService,
      * @throws EmailAlreadyUsedException {@code 400 (Bad Request)} if the email is already used.
      * @throws LoginAlreadyUsedException {@code 400 (Bad Request)} if the login is already used.
      */
+    @DisableEndpoint
```

Comment:
The reason I proposed the new name is at the place using this annotation, it's not so clear what it is for.

Reply:
I agree that the name makes it clearer. Will make that change.